bookwyrm/migrations/0190_book_search_updates.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2.20 on 2023-11-24 17:11
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("bookwyrm", "0188_theme_loads"),
```

Comment:
Could you rebase this PR on the latest `main`?

Reply:
(I can also do this if you'd like)

Reply:
Thanks, I performed a merge (just a preference), and produced 0196_merge_pr3134_into_main.py to take care of leaf migrations.